config/install/views.view.print_article.yml

```diff
@@ -0,0 +1,745 @@
+langcode: en
```

Comment:
An empty text, like on admin/content would be nice here.
![Screen empty print article list](http://share.undpaul.de/bilder/jh/Print_article__Site-Install_2017-04-24_06-06-49--zz9n8.png)